meta-sensaas.py

```diff
@@ -49,9 +49,6 @@
 if(whichexe in platform):
     # linux
     sensaasexe=sensaasexe+ "sensaas.py"
-elif platform == "darwin":
-    # OS X - linux version?
-    sensaasexe=sensaasexe+ "sensaas.py"
 else:
     #windows
     sensaasexe="python " + sensaasexe+ "sensaas.py"
```

Comment:
I've got the same issue on `linux` as well. Maybe it would be better to always execute `sensaas.py` via `python`?

```
sensaasexe = "python " + sensaasexe + "sensaas.py"
```

Reply:
Hi,
It can fail when there are several versions of Python and when the default 'python' is linked to another version than 3.7.
Not obvious to find what works on all systems!

Reply:
Hi @douguet , I completely agree it is tricky to find what works on all systems. What I was suggesting here is to always call `sensaas.py` via the Python interpreter, so that it shifts the responsibility to the user to ensure the correct interpreter (with the correct libraries installed) is available (for example via `virtualenv` or `conda`). In such case, provided that `python` maps to the correct interpreter, it _should_ work on all systems.